mail_forwarding/models/res_user.py

```diff
@@ -0,0 +1,14 @@
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    forwarding_user_id = fields.Many2one("res.users", string="Forwarding User")
```

Comment:
Thought it would allow more flexibility (i.e. forwarding emails to partners with no login) if the field was added to res.partner instead.

Reply:
we haven't think on a use case for this, but indeed it would be more flexible.
We could add it on partners but allow configuration from res.users and res.partner views.
It seem ok for you?

Reply:
Changes done @yostashiro . Now the functionality can be used on partners